examples/custom-config/values.yaml

```diff
@@ -11,6 +11,9 @@ externalServices:
     externalLabels:
       region: southwest
       tenant: widgetco
+    externalLabelsFrom:
+      env: remote.kubernetes.configmap.cluster_info.data["env"]
+      region: remote.kubernetes.configmap.cluster_info.data["region"]
```

Comment:
Can you update this example's README.md with the updated values file? Eventually, I want a script that syncs the two, or at least warns if they're out of sync.